package.json

```diff
@@ -36,7 +36,7 @@
     "fixmodulenotdefined": "yarn cache clean && cd core/client && rm -rf node_modules tmp dist && yarn && cd ../../"
   },
   "engines": {
-    "node": "^10.13.0 || ^12.10.0",
+    "node": "^10.13.0 || ^12.10.0 || ^14.13.0",
```

Comment:
@acburdine Would we be able to get rid of this? Given we have `--ignore-engines`, we don't need it for tests and I think having it gives the impression we're now Node 14 compatible (soon though!) :slightly_smiling_face:

Reply:
the issue I was running into with `--ignore-engines` is that when you run something like `yarn run <script>`, and then said script runs `yarn <something>` - the inner script fails due to engine failures, even when you pass `--ignore-engines`.

So if I ran `yarn lint --ignore-engines`, that in turn calls `yarn lint:server`, which fails because of the missing ignore-engines flag 😓